### PR TITLE
File Manager - Support context menu for folders tree view

### PR DESCRIPTION
**Pull request by vladkovl**
Key functionalities:
- context menu for files tree view
- tests refactoring - common code extracted to the separate `fileManagerHelpers.js` module

### CODE REVIEW

File: js/ui/file_manager/ui.file_manager.command_manager.js [hunk 1]
```diff
@@ -107,7 +100,7 @@ export class FileManagerCommandManager {
         if(!command || !command.enabled) {
             return false;
         }
-        return command.noFileItemRequired || items.length > 0 && (!command.isSingleFileItemCommand || items.length === 1);
+        return command.noFileItemRequired || items && items.length > 0 && (!command.isSingleFileItemCommand || items.length === 1);
```
Comment (DokaRus) [inline js/ui/file_manager/ui.file_manager.command_manager.js]:
```suggestion
        const itemsLength = items && items.length || 0;
        return command.noFileItemRequired || itemsLength > 0 && (!command.isSingleFileItemCommand || itemsLength === 1);
```

File: js/ui/file_manager/ui.file_manager.context_menu.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                }
+                result.push(item);
+            }
+        });
+
+        return result;
+    }
+
+    _createMenuItemByCommandName(commandName) {
+        const command = this._commandManager.getCommandByName(commandName);
```
Comment (DokaRus) [inline js/ui/file_manager/ui.file_manager.context_menu.js]:
```suggestion
        const { text, icon} = this._commandManager.getCommandByName(commandName);
```

File: js/ui/file_manager/ui.file_manager.context_menu.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        return {
+            commandName,
+            text: command.text,
```
Comment (DokaRus) [inline js/ui/file_manager/ui.file_manager.context_menu.js]:
```suggestion
            text,
```

File: js/ui/file_manager/ui.file_manager.context_menu.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            icon: command.icon,
```
Comment (DokaRus) [inline js/ui/file_manager/ui.file_manager.context_menu.js]:
```suggestion
            icon,
```

**Review by maxerokh — APPROVED** [review]
We are discussed personally with @vladkovl about this PR

**Pull request merged**